aclk/aclk_util.h

```diff
@@ -5,6 +5,19 @@
 #include "libnetdata/libnetdata.h"
 #include "mqtt_wss_client.h"
 
+#define CLOUD_EC_MALFORMED_NODE_ID   1
+#define CLOUD_EMSG_MALFORMED_NODE_ID "URL requests node_id but there is not enough chars following (for it to be valid uuid)."
+#define CLOUD_EC_NODE_NOT_FOUD       2
+#define CLOUD_EMSG_NODE_NOT_FOUD     "Node with requested node_id not found"
```

Comment:
`FOUD` begs for a spellcheck sooner or later :smile: Also this message should probably be just "Node id not found"  or "Node does not exist"

Reply:
thanks